src/styles/blocks/catalog.scss

```diff
@@ -16,10 +18,15 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $margin-catalog;
+    transition: all $transition-time $transition;
```

Comment:
it's not a good idea to use all as a value for transition-property. Specify what you'd like to change exactly